src/soot/SootMethodRefImpl.java

```diff
@@ -152,6 +163,11 @@ private SootMethod checkStatic(SootMethod ret) {
 	}
 
 	private SootMethod tryResolve(StringBuffer trace) {
+
+		if(isSignaturePolymorphic()) {
+			return new SootMethod(name(), parameterTypes(), returnType(), Modifier.PUBLIC);
```

Comment:
Please don't create SootMethods out of thin air. This method won't be registered in the Scene, will have no parent class, and will therefore violate various reasonable assumptions held by analysis clients.